packages/tooltips/src/elements/Tooltip.tsx

```diff
@@ -141,20 +177,12 @@ const Tooltip = ({
 };
 
 Tooltip.propTypes = {
-  /** Appends the tooltip to the body element */
-  appendToBody: PropTypes.bool,
-  arrow: PropTypes.bool,
-  children: PropTypes.node,
-  /** Milliseconds of delay before open/close of tooltip is initiated */
+  appendToNode: PropTypes.any,
```

Comment:
Was there a particular reason this prop type is `PropTypes.any` rather than `PropTypes.node`?

Reply:
The `createPortal()` definition takes an `Element` which isn't a subset of `PropTypes.node`. `any` is required here.

Reply:
Awesome, thanks for clarifying!